examples/fastai/READMD.md

```diff
@@ -0,0 +1,42 @@
+### MLflow Fastai Example
+
+This simple example illustrates how you can use `mlflow.fastai.autolog()` API
```

Comment:
```suggestion
This simple example illustrates how you can use the `mlflow.fastai.autolog()` API
```

Reply:
Done!
Ah. the file should be called README.md (not READMD.md) :-)